### ISSUE TITLE
xdis version

### ISSUE THREAD

**Issue by mowenli** [rocky/python-uncompyle6#462]
Both https://github.com/rocky/python-xdis and https://pypi.org/project/xdis/ describe that the latest version of xdis is 6.0.5. However, the requirement specified in 'pkginfo.py' is to install xdis<6.2.0,>=6.0.8. How can i install it?

**Comment by rocky** [comment]
If you have to ask, you can't ;-) 

Other than post here, what have you tried to solve this problem? 

StackOverflow or some other social medium is a better place for asking questions like this.

**Comment by mowenli** [comment]
Okay, let me rephrase the question. When I use the command 'python setup.py install', I encounter an issue where xdis cannot be installed. The error message states: 'Searching for xdis<6.2.0,>=6.0.8 Reading https://pypi.org/simple/xdis/ No local packages or working download links found for xdis<6.2.0,>=6.0.8'. How can I resolve this problem?

**Comment by firefart** [comment]
same here:

```text
#50 1.405 Processing dependencies for uncompyle6==3.9.1.dev0
#50 1.407 Searching for xdis<6.2.0,>=6.0.8
#50 1.407 Reading https://pypi.org/simple/xdis/
#50 4.352 No local packages or working download links found for xdis<6.2.0,>=6.0.8
#50 4.352 error: Could not find suitable distribution for Requirement.parse('xdis<6.2.0,>=6.0.8')
#50 ERROR: process "/bin/sh -c git clone --depth 1 https://github.com/rocky/python-uncompyle6.git /opt/uncompyle6 &&   cd /opt/uncompyle6 &&   python3 setup.py install" did not complete successfully: exit code: 1
```

**Comment by firefart** [comment]
looks like this commit referenced the non existing version https://github.com/rocky/python-uncompyle6/commit/1c28bc1c824d054d8ffa6994264d5724b147ab6b

**Comment by JacekJagosz** [comment]
All you need is to install xdis from source first, and then uncompyle6. Currently xdis from git advertises as `6.1.0.dev0`

**Comment by rocky** [comment]
Thanks @JacekJagosz for assisting the users.
